src/server/pkg/worker/api_server.go

```diff
@@ -190,6 +190,9 @@ func NewAPIServer(pachClient *client.APIClient, etcdClient *etcd.Client, etcdPre
 }
 
 func (a *APIServer) downloadData(logger *taggedLogger, inputs []*Input, puller *filesync.Puller, parentTag *pfs.Tag) error {
+	defer func(start time.Time) {
+		logger.Logf("download for inputs (%v) took %v\n", inputs, time.Since(start))
```

Comment:
No need to log the inputs here, the logger already has inputs present in a structured way.